deploy/consts.ts

```diff
@@ -270,5 +278,5 @@ export const CIRCLE_DOMAIN_IDs: { [chainId: number]: number } = {
   420: 2, // Optimism Goerli
   421613: 3, // Arbitrum Goerli
   84531: 6, // Base Goerli
-  80001: 7, // Polygon Mumbai
+  80002: 7, // Polygon Amoy
```

Comment:
Nit: we may want to keep the entry here (like we do with eth goerli & base goerli) - we should also probably add eth sepolia